pandas/tests/series/test_constructors.py

```diff
@@ -127,37 +127,40 @@ def test_constructor_empty(self, input_class):
         with tm.assert_produces_warning(DeprecationWarning, check_stacklevel=False):
             empty = Series()
             empty2 = Series(input_class())
-
-        # these are Index() and RangeIndex() which don't compare type equal
-        # but are just .equals
-        tm.assert_series_equal(empty, empty2, check_index_type=False)
+        tm.assert_series_equal(empty, empty2)
+        assert type(empty.index) is Index
```

Comment:
you can use isinstance

Reply:
I want to differentiate between `Index` and `RangeIndex`. The `isinstance` would be true for both of them